style.css

```diff
@@ -0,0 +1,66 @@
+
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+a{
+    font-weight: 500;
+}
+.container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    max-width: 1440px;
+    height: 569px;
+    background-color: #FFFFFF;
+}
+h2{
+    margin-top: 120px;
+    font-size: 40px;
+    font-weight: 500;
+    text-align: center;
+}
+.container>hr{
+    margin-top: 20px;
+    width: 100px;
+    height: 4px;
+    background-color: #F5BD41;
+}
+.wrapper{
+    margin-top: 82px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-left: 104px;
+    margin-right: 104px;
+}
+.reviews,.booking,.communication,.support{
+    width: 296px;
+    height: 176px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+
+}
+.wrapper_img{
+    width: 100px;
+    height: 100px;
+}
+.support>img{
+    width: 75px;
+    height: 92px;
+}
+.communication>img{
+    width: 83px;
+    height: 83px;
```

Comment:
you shouldn't use diffrent class names and diffrent sizes for images, you've downloaded wrong images, please choose one size images, and use one class name